gen/src/generate/idna/shared/version.rs

```diff
@@ -0,0 +1,55 @@
+// TODO: Unify this with generate::ucd::shared::version
```

Comment:
The reason the data files are explicitly under `<component>/src/` is to have them *included* in packaging. If we move them out of the `<component>` dir, like `../shared/`, then the standalone packages won't build from `.crate` file.

Or maybe I misunderstood what you meant here.